resources/conf/config.toml

```diff
@@ -15,6 +15,16 @@ listenerTLSEnabled = true
 listenerCertPath = "/home/wso2/security/localhost.pem"
 listenerKeyPath = "/home/wso2/security/localhost.key"
 
+[envoy.upstream]
+[envoy.upstream.tls]
+minimumProtocolVersion = "TLS1_2"
+maximumProtocolVersion = "TLS1_2"
+ciphers = "ECDHE-ECDSA-AES128-GCM-SHA256, ECDHE-RSA-AES128-GCM-SHA256, ECDHE-ECDSA-AES128-SHA, ECDHE-RSA-AES128-SHA, AES128-GCM-SHA256, AES128-SHA, ECDHE-ECDSA-AES256-GCM-SHA384, ECDHE-RSA-AES256-GCM-SHA384, ECDHE-ECDSA-AES256-SHA, ECDHE-RSA-AES256-SHA, AES256-GCM-SHA384, AES256-SHA"  
```

Comment:
Check against unavailable cipher suites.

Reply:
When an unavailable cipher suite is set in the server configuration (backend), the following error is received. 
```
< HTTP/1.1 503 Service Unavailable
< content-length: 196
< content-type: text/plain
< date: Tue, 05 Jan 2021 19:13:38 GMT
< server: envoy
<
* Connection #0 to host localhost left intact
upstream connect error or disconnect/reset before headers. reset reason: connection failure, transport failure reason: TLS error: 268436536:SSL routines:OPENSSL_internal:TLSV1_ALERT_INTERNAL_ERROR* Closing connection 0
```